RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -696,18 +696,18 @@ void Event::createTrackCandidates(bool no_pls_dupclean, bool tc_pls_triplets) {
                       tc_pls_triplets);
 
   // Check if either n_max_pixel_track_candidates or n_max_nonpixel_track_candidates was reached
-  auto nTrackCanpT5Host_buf = allocBufWrapper<unsigned int>(cms::alpakatools::host(), 1, queue_);
-  auto nTrackCanpT3Host_buf = allocBufWrapper<unsigned int>(cms::alpakatools::host(), 1, queue_);
-  auto nTrackCanpLSHost_buf = allocBufWrapper<unsigned int>(cms::alpakatools::host(), 1, queue_);
-  auto nTrackCanT5Host_buf = allocBufWrapper<unsigned int>(cms::alpakatools::host(), 1, queue_);
+  auto nTrackCanpT5Host_buf = cms::alpakatools::make_host_buffer<unsigned int[]>(queue_, 1u);
```

Comment:
For scalars, we could use
```
cms::alpakatools::make_host_buffer<unsigned int>(queue_);
```

The issue is that I couldn't find an easy way to create a scalar view for the scalars in SoAs, so `memcpy` fails to compile since they have different dimensions.

Reply:
Actually, I figured out how to do this